package.json

```diff
@@ -75,9 +75,10 @@
     "postbuild": "npm-run-all --parallel copy:*",
     "postinstall": "node postinstall.js",
     "start": "node build/server.js",
-    "test": "npm run test:server",
+    "test": "npm run test:server; npm run test:client",
```

Comment:
At least, you should do `npm run test:server && npm run test:client`. Else, if the `npm run test:server` fails but not the `npm run test:client`, the return code will still be OK and the failure can be easily missed.